src/item.cpp

```diff
@@ -3965,6 +3984,19 @@ bool item::ammo_consume( int qty ) {
     return false;
 }
 
+itype_id item::ammo_current() const
+{
+    if( is_ammo() ) {
+        return typeId();
```

Comment:
Documentation says "Specific ammo type, returns "null" if item is neither ammo nor loaded with any", but this returns an empty string instead.